### ISSUE TITLE
Remote access isn't working

### ISSUE THREAD

**Issue by uqlel** [WebThingsIO/gateway#2817]
Hello,
After installing webthings gateway on raspbian using .deb file and setuping, when i try to connect to my domain, Firefox throws SSL_ERROR_ACCESS_DENIED_ALERT error. 
`grep -i pagekite run-app.log` shows this:
```bash
2021-04-23 10:56:30.525 INFO   : [pagekite] stderr: /bin/sh: 1: /opt/webthings-gateway/pagekite.py: not found
2021-04-23 10:56:30.532 INFO   : [pagekite] process exited with code 127
2021-04-23 11:04:05.528 INFO   : [pagekite] stderr: /bin/sh: 1: /opt/webthings-gateway/pagekite.py: not found
2021-04-23 11:04:05.533 INFO   : [pagekite] process exited with code 127
2021-04-23 11:10:45.770 INFO   : [pagekite] stderr: /bin/sh: 1: /opt/webthings-gateway/pagekite.py: not found
2021-04-23 11:10:45.774 INFO   : [pagekite] process exited with code 127
```
Sorry for my bad english :/

**Comment by uqlel** [comment]
I can detect pagekite.net:
```bash
root@rpi:/etc/webthings-gateway/log# ping pagekite.net
PING pagekite.net (69.164.211.158) 56(84) bytes of data.
64 bytes from ns1.pagekite.net (69.164.211.158): icmp_seq=1 ttl=53 time=102 ms
64 bytes from ns1.pagekite.net (69.164.211.158): icmp_seq=2 ttl=53 time=100 ms
64 bytes from ns1.pagekite.net (69.164.211.158): icmp_seq=3 ttl=53 time=100 ms
^C
--- pagekite.net ping statistics ---
3 packets transmitted, 3 received, 0% packet loss, time 2ms
rtt min/avg/max/mdev = 100.049/100.675/101.603/0.762 ms```

**Comment by uqlel** [comment]
I have also changed `--inh-caps=-all` to `--inh-caps=-chown` in `/usr/bin/webthings-gateway` because i had the same issue as in https://github.com/WebThingsIO/gateway-aur/issues/4

**Comment by uqlel** [comment]
`find / -name "*pagekite*"` returns nothing, i think that i'm missing something

**Comment by benfrancis** [comment]
I'm afraid I've never actually used the .deb package so I don't have much experience with that and it's currently lacking a maintainer, but what I can say is:
- You shouldn't need to access pagekite.net
- You do need pagekite.py to be present on the gateway in order for the tunnel to work (see https://github.com/WebThingsIO/gateway/blob/master/pagekite.py)

If the PageKite script is indeed not present in the .deb package then that would be a bug.

**Comment by benfrancis** [comment]
@uqlel I've just discovered that this bug is reproduceable when building from source too, on systems where python2 is not installed (because PageKite depends on python2).

From what I can tell python2 should have been pulled in with the python-six dependency in the .deb package (assuming the Raspbian package has the same dependencies as the Debian package), but it doesn't seem like that happened in your case.

Would you mind installing python2 (sudo apt install python2) and trying again to see whether that fixes the problem? If it does then we can look at why python2 isn't being installed as a dependency.

**Comment by benfrancis** [comment]
In the meantime I submitted a PR (#2853) to tweak the dependency list when building from source.

**Comment by yuchingtw** [comment]
Hi, @benfrancis , I found that Pagekite seems to support python3 after v1.5, maybe we don't need to use python2 anymore?

reference: https://pagekite.net/2019-02-25/Pagekite_py_v1_0_released

**Comment by benfrancis** [comment]
@yuchingtw That's interesting. Have you tested it? IIRC it wasn't working for me and we're already using version 1.5.2.201011.

**Comment by yuchingtw** [comment]
I tested it on Ubuntu 20.04, using Python 3.8.10 (Python2 is not installed by default), and the version of pagekite is 1.5.2.201011. Then running pagekite according to the parameters on line 112 in tunnel-service.ts can run successfully.

In addition, it was found that if Python2 is not installed, the following error will occur in the gateway:

```
2021-09-24 20:10:49.861 INFO: [pagekite] stderr: /bin/sh: 1: /home/yuching/gateway/pagekite.py: not found
2021-09-24 20:10:49.862 INFO: [pagekite] process exited with code 127
```

Then I checked that pagekite.py exists, so I thought it may be `pagekite_cmd: path.normalize(path.join(process.cwd(),'pagekite.py'))` in config/default.js directly execute pagekite by `./pagekite.py`, but if it is changed to `python3 pagekite.py` will be successful.

**Comment by benfrancis** [comment]
Yes I think the problem may be that the [shebang](https://github.com/WebThingsIO/gateway/blob/master/pagekite.py#L1) (first line) in pagekite.py is `#!/usr/bin/python` rather than `#!/usr/bin/python3`, but I don't want to just change it in case it isn't supported on all systems. Is there a newer version of PageKite which explicitly uses the python3 binary?

**Comment by yuchingtw** [comment]
I have looked for [PageKite's GitHub](https://github.com/pagekite/PyPagekite) but found that the latest version seems to be the 1.5.2.201011 which we are using, and there is no version that explicitly uses the python3 binary.

But in [PEP](https://www.python.org/dev/peps/pep-0394/#for-python-runtime-distributors), it is recommended to specify the version in the shebang. Maybe we can consider modifying the shebang directly for python3.

**Comment by nylki** [comment]
> I've changed --inh-caps=-all to --inh-caps=-chown in /usr/bin/webthings-gateway because i had the same issue as in WebThingsIO/gateway-aur#4

Just wanted to confirm that, I had to do the same thing on a rpi3 .deb package install.
